config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -25,6 +25,9 @@ patches:
       - name: "kata-qemu"
         snapshotter: "nydus"
         pulltype: ""
+      - name: "kata-qemu-coco-dev"
```

Comment:
Hey, this was on my TODO list, thanks!

I plan to create a job to test that runtimeclass too: https://github.com/confidential-containers/operator/issues/409 . But let me know if you are already working on that.

Reply:
I am not working on that, just had to add this for a personal test earlier Today, and shared as I had that done already.

Reply:
@fidencio might I ask why on TEE you want nydus and on non-TEE you don't? Also you mentioned in kata there are the defaults, is it possible to rely on those defaults and not setting it here? (would empty string result in using the default or is this not technically possible?)